fluss-server/src/main/java/com/alibaba/fluss/server/zk/data/TableRegistration.java

```diff
@@ -69,24 +80,39 @@ public TableDescriptor toTableDescriptor(Schema schema) {
                 TableDescriptor.builder()
```

Comment:
The method `toTableDescriptor` can be removed?

Reply:
good catch!